src/__integrationtests__/utils.ts

```diff
@@ -5,6 +5,9 @@ import Identity from '../identity/Identity'
 import CType from '../ctype/CType'
 import ICType from '../types/CType'
 import { getOwner } from '../ctype/CType.chain'
+import { DEFAULT_WS_ADDRESS } from '../blockchainApiConnection'
+
+export const WS_HOST = process.env.WS_HOST || DEFAULT_WS_ADDRESS
```

Comment:
A nice alternative would be to make the global default configurable via an environment variable by changing line 16 on BlockchainApiConnection.ts to something like
```typescript
export const DEFAULT_WS_ADDRESS = process.env.DEFAULT_WS_ADDRESS || 'ws://127.0.0.1:9944'
```

Reply:
@tjwelde Is this a security relevant change? Would it be potentially dangerous to have the default set via environment variable, because it opens up new ways to have the sdk connect to a different endpoint?
It would allow me to revert many of the proposed changes on the tests, so I'd prefer this way of handling endpoint configuration.

Reply:
implemented this now, lemme know if you think that's a problem

Reply:
Looks good